examples/_d2n-std/nix/app/packages.nix

```diff
@@ -0,0 +1,15 @@
+{
+  inputs,
+  cell,
+}: {
+  app = {
+    source = inputs.src;
+    projects = {
+      prettier = {
+        name = "prettier";
+        subsystem = "nodejs";
+        translator = "yarn-lock";
+      };
+    };
+  };
```

Comment:
Help, please! :pray: &mdash; This needs to be a `derivation`. How would I do that?